nativelib/src/main/scala/scala/scalanative/runtime/CAtomics/CAtomicInt.scala

```diff
@@ -0,0 +1,87 @@
+package scala.scalanative
+package runtime.CAtomics
+
+import scala.scalanative.native.{CInt, Ptr, sizeof, stackalloc}
+import scala.scalanative.runtime.Atomic
+import scala.scalanative.runtime.Atomic._
+
+class CAtomicInt extends CAtomic {
```

Comment:
Yeah you're right, it's an artifact of the old code I was trying to use I'll change that